pkg/iac-providers/terraform/commons/load-dir.go

```diff
@@ -344,6 +347,15 @@ func (t TerraformDirectoryLoader) buildUnifiedConfig(rootMod *hclConfigs.Module,
 				}
 			}
 
+			// verify whether the module source directory has any .tf config files
+			if !t.parser.IsConfigDir(pathToModule) {
```

Comment:
we should modify this check because this will also modify the error message for non existing directories.